spec/controllers/synapses_controller_spec.rb

```diff
@@ -35,13 +35,30 @@
       end
     end
 
+    context 'with private topics' do
+      it 'redirects to /' do
```

Comment:
Yes, true. And actually, I am thinking that in cases like synapse creation, itd be good to start using our own API :) then just the main app handle page loads and special functions